components/cdc/src/endpoint.rs

```diff
@@ -549,30 +552,38 @@ impl<T: 'static + RaftStoreRouter<RocksEngine>> Endpoint<T> {
                 error!("cdc schedule cdc task failed"; "error" => ?e);
             }
         };
+        let raft_router = self.raft_router.clone();
         let scheduler = self.scheduler.clone();
-        if let Err(e) = self.raft_router.significant_send(
-            region_id,
-            SignificantMsg::CaptureChange {
-                cmd: change_cmd,
-                region_epoch: request.take_region_epoch(),
-                callback: Callback::Read(Box::new(move |resp| {
-                    if let Err(e) = scheduler.schedule(Task::InitDownstream {
-                        downstream_id,
-                        downstream_state,
-                        cb: Box::new(move || {
-                            cb(resp);
-                        }),
-                    }) {
-                        error!("cdc schedule cdc task failed"; "error" => ?e);
-                    }
-                })),
-            },
-        ) {
-            warn!("cdc send capture change cmd failed"; "region_id" => region_id, "error" => ?e);
-            deregister_downstream(Error::request(e.into()));
-            return;
-        }
+        let scan_concurrency_semaphore = self.scan_concurrency_semaphore.clone();
         self.workers.spawn(async move {
+            let _permit = scan_concurrency_semaphore.acquire().await;
+
```

Comment:
Should we test for request cancellation here? If not, what would happen if the client sends 10000 requests and cancels the connection immediately, and then another client retries? Would the second client encounter a large latency to receiving data? Is this behavior risky?